packages/glob/__tests__/internal-globber.test.ts

```diff
@@ -255,6 +255,7 @@ describe('globber', () => {
         'file-under-a'
       )
     ])
+    await unlinkSymlinkDir(itemPaths)
```

Comment:
Adding an unlink here due to [mac specific `ENAMETOOLONG` errors](https://github.com/eslint/eslint/pull/15865) caused by the linter.